src/components/search/content-highlights/tests/HighlightedContentCard.test.jsx

```diff
@@ -1,5 +1,5 @@
 import { AppContext } from '@edx/frontend-platform/react';
-import { shallow } from 'enzyme';
+import { shallow } from '@edx/react-unit-test-utils';
```

Comment:
[suggestion] Would recommend not using `@edx/react-unit-test-utils` here in favor of refactoring the one test using `shallow` to use RTL directly instead;`shallow` rendering is not something we want to continue as a paradigm in this MFE.

https://buttondown.email/kentcdodds/archive/76ce5bb1-0681-490b-9e1e-62f27e5cfa4d